### PR TITLE
Add Setup page

### PR DESCRIPTION
**Pull request by CitMC**
Update to include a setup page at the beginning of module "Running Your Own Cosmos Chain - Part 1".

### Change scope

The changes in this Pull Request include (please tick all that apply):

- [ ] Small language/grammar fixes
- [ ] Small content fixes 
- [x] Addition of new content
- [ ] Sample code/command updates
- [ ] Platform fixes
- [ ] Other

### CODE REVIEW

**Comment by CitMC** [comment]
@coldice please merge.

File: academy/setup.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+</HighlightBox>
+
+## Install Rust
+
+To work with Rust, you may need to install it first. You can find a well-documented install step-by-step in the [Install page from Rust](https://www.rust-lang.org/tools/install).
+
+To check that you are up-to-date run:
+
+```
+rustc --version
```
Comment (xavierlepretre) [inline academy/setup.md:57]:
```suggestion
$ rustc --version
```

File: academy/setup.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+If you already installed the Rust toolchain, you can update your installation by running:
+
+```
+rustup update
```
Comment (xavierlepretre) [inline academy/setup.md:63]:
```suggestion
$ rustup update
```

File: academy/setup.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+```
+
+To test the installation of Rust, you can use the following command:
+
+```
+cargo version
```
Comment (xavierlepretre) [inline academy/setup.md:69]:
```suggestion
$ cargo version
```

**Review by coldice — APPROVED** [review]

**Pull request merged**